src/test/java/com/google/devtools/build/lib/sandbox/cgroups/CpuTest.java

```diff
@@ -52,16 +52,18 @@ public void getCpuLimit_v1() throws IOException {
 
   @Test
   public void setCpuLimit_v2() throws IOException {
-    File limit = scratch.file("cgroup/cpu/cpu.max", "-1 100000").getPathFile();
+    File limit = scratch.file("cgroup/cpu/cpu.max", "-1 100000\n").getPathFile();
     Cpu cpu = new UnifiedCpu(scratch.path("cgroup/cpu").getPathFile().toPath());
     cpu.setCpus(5);
     assertThat(Files.asCharSource(limit, UTF_8).read()).isEqualTo("500000 100000");
   }
 
   @Test
   public void getCpuLimit_v2() throws IOException {
-    scratch.file("cgroup/cpu/cpu.max", "6000 1000");
+    scratch.file("cgroup/cpu/cpu.max", "6000 1000\n");
     Cpu memory = new UnifiedCpu(scratch.path("cgroup/cpu").getPathFile().toPath());
     assertThat(memory.getCpus()).isEqualTo(6);
+    scratch.file("cgroup/cpu/cpu.max", "max 1000\n");
+    assertThat(memory.getCpus()).isEqualTo(Runtime.getRuntime().availableProcessors());
```

Comment:
Would you please move the `max 1000` case to a separate test?

Reply:
Can you also help us fix the naming here? I suppose the variable name should be `cpu` instead of `memory`.

Reply:
Done and done!